packages/qt5-qtbase/build.sh

```diff
@@ -101,20 +100,20 @@ termux_step_configure () {
         -no-gbm \
         -no-kms \
         -no-linuxfb \
-        -no-mirclient \
         -no-libudev \
         -no-evdev \
         -no-libinput \
         -no-mtdev \
         -no-tslib \
-        -system-xcb \
-        -no-xcb-xinput \
```

Comment:
`-no-xcb-xinput` is renamed to `-no-bundled-xcb-xinput` in Qt 5.15.

Without this option I expect some weird behavior described in commit 171e655d1fd6a3eeacffde46fb7a46fcd6be233b.

Reply:
I'm confused. Should this rather be `-bundled-xcb-xinput`, which as an effect disables `system-xcb-xinput`?

It seems that we have to enable either `bundled-xcb-xinput` or `system-xcb-xinput` but cannot disable both of them.

Reply:
> `-no-xcb-xinput` is renamed to `-no-bundled-xcb-xinput` in Qt 5.15.
> 
> Without this option I expect some weird behavior described in commit [171e655](https://github.com/termux/x11-packages/commit/171e655d1fd6a3eeacffde46fb7a46fcd6be233b).

I guess we have to test it for Qt 5.15?